sql/ddl_triggers.sql

```diff
@@ -1,5 +1,6 @@
 CREATE OR REPLACE FUNCTION _timescaledb_internal.ddl_command_end() RETURNS event_trigger
 AS '$libdir/timescaledb', 'timescaledb_ddl_command_end' LANGUAGE C IMMUTABLE STRICT;
 
+DROP EVENT TRIGGER IF EXISTS timescaledb_ddl_command_end;
```

Comment:
I'd prefer if this is at least a separate commit, with proper commit message explaining why this is needed and what it fixes. Now this gets lost in the "prepare" commit that should really be routine changes.